### ISSUE TITLE
Build.PL fails without Alien::bison

### ISSUE THREAD

**Issue by chorny** [selectiveintellect/p5-alien-gputils#1]
```
$ perl Build.PL 
Checking prerequisites...
  build_requires:
    !  Alien::bison is not installed

ERRORS/WARNINGS FOUND IN PREREQUISITES.  You may wish to install the versions
of the modules indicated above before proceeding with this installation

Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Alien-gputils' version '0.07'
Created META.yml and META.json
File 'MANIFEST.SKIP' does not exist: Creating a temporary 'MANIFEST.SKIP'
Building Alien-gputils
Downloading File: gputils-1.4.0-1.tar.gz ... Done
Extracting Archive ... Done
Building library ... 
+ cd /home/c/.cpan/build/Alien-gputils-0.07-ulEopR/_alien/gputils-1.4.0-1
Can't locate Alien/bison.pm in @INC (you may need to install the Alien::bison module) (@INC contains: /home/c/.cpan/build/Alien-gputils-0.07-ulEopR/_build/lib /home/c/perl5/perlbrew/perls/18.1i/lib/site_perl/5.18.1/i686-linux-thread-multi /home/c/perl5/perlbrew/perls/18.1i/lib/site_perl/5.18.1 /home/c/perl5/perlbrew/perls/18.1i/lib/5.18.1/i686-linux-thread-multi /home/c/perl5/perlbrew/perls/18.1i/lib/5.18.1 .) at (eval 165) line 1.
BEGIN failed--compilation aborted at (eval 165) line 1.
```

So it should be in configure_requires.

**Comment by vikasnkumar** [comment]
@chorny  what OS is this running on ? On Linux it should force Alien::Bison as a requisite.

**Comment by vikasnkumar** [comment]
Added it and released to CPAN as version 0.08

**Comment by chorny** [comment]
Prerequisites from build_requires are installed after Build.PL is run. If you use something in Build.PL, you need to use configure_requires.
